pkglistgen.py

```diff
@@ -1233,6 +1235,12 @@ def update_and_solve_target(self, api, target_project, target_config, main_repo,
         product = target_config.get('pkglistgen-product', '000product')
         release = target_config.get('pkglistgen-release', '000release-packages')
 
+        opts.filtered_architectures = []
```

Comment:
Could use python set intersection like `repo_checker`, but obviously this works fine as well.

https://github.com/openSUSE/openSUSE-release-tools/blob/39a3f83180e0e54658454da91662d1eabbba2c4e/repo_checker.py#L109-L118